### PR TITLE
Feat: Added Auth Pages

### PR DESCRIPTION
**Pull request by rittik-ghosh**
Hi,

I have added the following pages :
1) Signup
2) Login
3) Logout

Please review and let me know of any improvements.

login auth
signup auth
logout auth

### CODE REVIEW

File: Routes/Authentication/login.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Shoe-kart | Login</title>
+    <link rel="stylesheet" href="/Stylesheets/main.css">
+    <link rel="stylesheet" href="https://maximus-ui.netlify.app/Stylesheets/style.css">
```
Comment (aishwarya-bm) [inline Routes/Authentication/login.html:10]: this css can be extracted to main CSS file, this way you can avoid importing in all HTML files

File: Routes/Authentication/logout.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Shoe-kart | Logout</title>
+    <link rel="stylesheet" href="/Stylesheets/main.css">
+    <link rel="stylesheet" href="https://maximus-ui.netlify.app/Stylesheets/style.css">
```
Comment (aishwarya-bm) [inline Routes/Authentication/logout.html:10]: same as above

File: Routes/Authentication/signup.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Shoe-kart | Signup</title>
+    <link rel="stylesheet" href="/Stylesheets/main.css">
+    <link rel="stylesheet" href="https://maximus-ui.netlify.app/Stylesheets/style.css">
```
Comment (aishwarya-bm) [inline Routes/Authentication/signup.html:10]: same as above

File: Stylesheets/authentication.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
+.auth-main-container {
+    min-width: 33rem;
+    margin: auto;
+    position: absolute;
+    left: 43rem;
+    top: 16rem;
+    padding: 2rem;
+    border-radius: 4px;
```
Comment (aishwarya-bm) [inline Stylesheets/authentication.css:5]: from what i understand you are trying to center the login form, giving these values might affect when you resize. I think transform properties can help you better to center you form

File: Stylesheets/authentication.css [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    text-align: center;
+    margin-bottom: 2rem;
+    color: var(--st-primary-color);
+}
+
+.auth-input label{
+    font-style: normal;
+    font-weight: 1000;
+    font-size: 1.2rem;
+    line-height: 24px;
```
Comment (aishwarya-bm) [inline Stylesheets/authentication.css]: you can change this to rem for consistency

File: Stylesheets/authentication.css [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+.pswd-options{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```
Comment (aishwarya-bm) [inline Stylesheets/authentication.css:26]: you can have a util class for this, please use that if you have or in the interest of time add LATER

File: Stylesheets/authentication.css [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+.auth-btns .btn{
+    width: 100%;
+}
+
+.auth-link{
+    text-decoration: none;
+    font-weight: 1000;
+    font-size: 1.2rem;
+    line-height: 24px;
```
Comment (aishwarya-bm) [inline Stylesheets/authentication.css]: you can change px to rem

**Review by aishwarya-bm — COMMENTED** [review]
Overall code looks very clean with really good names for class. Class names imply the meaning very easily. Great work done here. Suggested few changes

File: Routes/Authentication/login.html [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+</head>
```
Comment (shuntzhere) [inline Routes/Authentication/login.html]: extra tag here.

File: Routes/Authentication/logout.html [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+</head>
```
Comment (shuntzhere) [inline Routes/Authentication/logout.html]: Same here.

File: Routes/Authentication/signup.html [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+</head>
```
Comment (shuntzhere) [inline Routes/Authentication/signup.html]: Here is an extra tag too.

File: Stylesheets/authentication.css (hunk 4 above)
Comment (shuntzhere) [reply Stylesheets/authentication.css:5]: Adding to what Aishwarya mentioned, You can also do margin 0 auto and remove the absolute positioning values. That would do centre the container as well.

File: Stylesheets/authentication.css [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+    left: 43rem;
+    top: 16rem;
+    padding: 2rem;
+    border-radius: 4px;
+}
+
+.auth-header h3 {
+    font-size: 2rem;
+    font-weight: 2000;
+    text-align: center;
```
Comment (shuntzhere) [inline Stylesheets/authentication.css]: Font-weight is a fixed value provided by the font providers. Check the weights you have imported and use them. I don't think 2000 font-weight exists.

File: Stylesheets/authentication.css [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+    margin-bottom: 2rem;
+    color: var(--st-primary-color);
+}
+
+.auth-input label{
+    font-style: normal;
+    font-weight: 1000;
+    font-size: 1.2rem;
```
Comment (shuntzhere) [inline Stylesheets/authentication.css]: Same here for font-weight and if possible use variables for weights.

**Review by shuntzhere — CHANGES_REQUESTED** [review]
Have suggested changes to be made. Also, do indent the code properly to increase the readability and remove the extra spaces.

File: Routes/Authentication/login.html (hunk 1 above)
Comment (rittik-ghosh) [reply Routes/Authentication/login.html:10]: thanks aishwarya, I will put them to the main.css

File: Routes/Authentication/login.html (hunk 8 above)
Comment (rittik-ghosh) [reply Routes/Authentication/login.html]: done. thanks for pointing it out

File: Routes/Authentication/signup.html (hunk 10 above)
Comment (rittik-ghosh) [reply Routes/Authentication/signup.html]: done. thanks for pointing it out

File: Routes/Authentication/logout.html (hunk 9 above)
Comment (rittik-ghosh) [reply Routes/Authentication/logout.html]: done. thanks for pointing it out

File: Stylesheets/authentication.css (hunk 12 above)
Comment (rittik-ghosh) [reply Stylesheets/authentication.css]: sure thanks

File: Stylesheets/authentication.css (hunk 11 above)
Comment (rittik-ghosh) [reply Stylesheets/authentication.css]: yes it doesn't exists. Thanks for pointing it out. I have changed it to 900

File: Stylesheets/authentication.css (hunk 6 above)
Comment (rittik-ghosh) [reply Stylesheets/authentication.css:26]: sure thanks for pointing it out

File: Stylesheets/authentication.css (hunk 4 above)
Comment (rittik-ghosh) [reply Stylesheets/authentication.css:5]: done have changed it to margin : auto

File: Stylesheets/authentication.css (hunk 7 above)
Comment (rittik-ghosh) [reply Stylesheets/authentication.css]: changed it to rems

File: Stylesheets/authentication.css (hunk 5 above)
Comment (rittik-ghosh) [reply Stylesheets/authentication.css]: done

**Pull request merged**